packages/flutter/test/cupertino/text_field_test.dart

```diff
@@ -1290,6 +1290,35 @@ void main() {
     expect(text.style.fontWeight, FontWeight.w400);
   });
 
+
+
+  testWidgets('text field toolbar options correctly changes options', (WidgetTester tester) async {
+    final TextEditingController controller = TextEditingController(
+      text: "j'aime la poutine",
+    );
+    await tester.pumpWidget(
+      CupertinoApp(
+        home: Column(
+          children: <Widget>[
+            CupertinoTextField(
+              controller: controller,
+              toolbarOptions: const ToolbarOptions(selectAll: true),
+            ),
+          ],
+        ),
+      ),
+    );
+
+    await tester.longPressAt(
+      tester.getTopRight(find.text("j'aime la poutine"))
+    );
+
+    await tester.pump();
+    await tester.pump(const Duration(milliseconds: 200));
+
+    // Sanity check that the toolbar widget exists and does not inclue [Select All].
+    expect(find.text('Select All'), findsOneWidget);
```

Comment:
can you also check for absent of Copy, Paste, Cut? you can use
```
expect(find.text('Copy'), findsNothing);
expect(find.text('Paste'), findsNothing);
expect(find.text('Cut'), findsNothing);
```

Reply:
done